### DISCUSSION TITLE
does it work with GraphQL?

### DISCUSSION THREAD

**Discussion by violabg** [ComfortablyCoding/strapi-plugin-transformer#6] [Q&A] [answered]
does it work with GraphQL?

**Comment by ComfortablyCoding** [comment, marked as answer]
Hello,

Currently this plugin does not work with GraphQL, but I am looking into adding support for it. 

I have managed to handle the reponse part and am currently investigating on how to modify the entity schemas to remove the attributes keys so no mismatch between the request and reponse.

**Reply by SalahAdDin** [reply]
And you couldn't make it work, right?

**Reply by ComfortablyCoding** [reply]
Correct, still haven't found a easy way to achieve it.
